.pylintrc

```diff
@@ -11,7 +11,7 @@
 #  See the License for the specific language governing permissions and
 #  limitations under the License.
 #
-[MASTER]
+[MAIN]
```

Comment:
Is this a necessary change ?

Reply:
The pylint 3.0 release notes state "Following a deprecation period, it's no longer possible to use MASTER or master as configuration section in setup.cfg or tox.ini". It seems that the section name in `.pylintrc` might actually be ignored, but the 3.x documentation exclusively mentions `MAIN`.

Reply:
Got it.